vowpalwabbit/distributionally_robust.h

```diff
@@ -29,6 +30,30 @@ namespace distributionally_robust
     {
     }
     double qfunc(double w, double r) { return unbounded ? 1 : -(gamma + (beta + r) * w) / ((n + 1) * kappa); }
+
+    void save_load_aml(io_buf& model_file, bool read, bool text)
```

Comment:
Why aml? Is this different or just a regular save/load function for DRO?

Reply:
```suggestion
    void save_load(io_buf& model_file, bool read, bool text)
```